src/commands/org/create/snapshot.ts

```diff
@@ -23,6 +23,7 @@ export class SnapshotCreate extends SfCommand<OrgSnapshot> {
   public static readonly examples = messages.getMessages('examples');
   public static readonly aliases = ['force:org:snapshot:create'];
   public static readonly deprecateAliases = true;
+  public static readonly state = 'closedPilot';
```

Comment:
plugin-command-reference checks this prop:
https://github.com/salesforcecli/plugin-command-reference/blob/e17f4039a704712b1df326ede80d5de520532e87/src/ditamap/command.ts#L100-L111

and oclif will show the state in the cmd help text.

Reply:
@cristiand391 -- almost there!  But here are two more things:

* (request) Can you change the message in the `--help` output from "This command is in closedPilot." to "This command is in closed Pilot"?  Without the space it looks a bit odd.
* The XML generated by plugin-command-reference is now adding the (Pilot) tag and note (yay), but the <xref> inside the <note> isn't picking up the Trailblazer link.  It looks like this (note the empty href):

`<xref href=""  format="html" scope="external" type="new-window:HTML"></xref>`

If the latter is an issue with plugin-command-ref, I can add a GUS item.

Reply:
> (request) Can you change the message in the --help output from "This command is in closedPilot." to "This command is in closed Pilot"? Without the space it looks a bit odd.

This is done in oclif: https://github.com/oclif/core/blob/main/src/help/index.ts#L149
I could add another msg in each command summary but that would show 2 messages (can't disable the one from oclif).

> The XML generated by plugin-command-reference is now adding the (Pilot) tag and note (yay), but the <xref> inside the <note> isn't picking up the Trailblazer link. It looks like this (note the empty href):

I think it's a bug, v2 was merged into main a few weeks so it might be that. Will check next week.